### PR TITLE
Add activation bars to sequence diagrams

### PR DESCRIPTION
**Pull request by NeoMinWei**

### CODE REVIEW

**Review by ryan1604 — APPROVED** [review]
LGTM

File: docs/diagrams/cashflow/AddRecurringSequence.puml [hunk 1]
```diff
@@ -7,18 +7,32 @@ participant ":Expense" as Expense
 
 loop current date is after or equals next recurring date
     LoadData -> Cashflow: setHasRecurred(true)
+    activate Cashflow
+    return
 
 alt income
     create Income
     LoadData -> Income: Income((Income) cashflow)
+    activate Income
```
Comment (wwweert123) [inline docs/diagrams/cashflow/AddRecurringSequence.puml:16]: Very nice. I will do this as well

**Review by wwweert123 — COMMENTED** [review]
LGTM

**Review by wwweert123 — APPROVED** [review]
good

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -491,3 +491,226 @@ shows you the welcome screen for the financial planner app
...
+You have removed an Expense
+   Type: Insurance
+   Amount: 800.00
+   Recurring every: 365 days, date added: Nov 12 2023, recurring on: Nov 11 2024
+   Description: insurance
+from the Financial Planner.
+Balance: 0.00
+```
+
+### Recurring cashflow
```
Comment (wwweert123) [inline docs/DeveloperGuide.md:693]: Please hellp me change my header

**Review by wwweert123 — APPROVED** [review]

**Review by ryan1604 — APPROVED** [review]
LGTM, dont need show output tho if not its just ug

**Pull request merged**